src/main/java/ua/danit/final_project/controllers/WorkShiftController.java

```diff
@@ -35,7 +35,7 @@ public WorkShiftController(WorkCommentService workCommentService) {
   public List<ShiftCommentDto> getByDate(@RequestParam(name = "date", required = false) Long millis) {
     return workCommentService.getShiftCommentsByDate(millis)
             .stream()
-            .map(ShiftCommentDto::new)
+            .map(shiftComment -> new ShiftCommentDto(shiftComment))
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) ([squid:S1612](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AS1612))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=105&issues=AWXJ1tSBlgbYnohY1fRL&open=AWXJ1tSBlgbYnohY1fRL)

[ISSUE_KEY]: AWXJ1tSBlgbYnohY1fRL